Features/foutafhandeling.feature

```diff
@@ -49,8 +49,7 @@ Bij valideren van een parameter tegen schema kunnen de volgende meldingen komen:
 | minimum          | Waarde is lager dan minimum {minimum}.                    | minimum      |
 | maximum          | Waarde is hoger dan maximum {maximum}.                    | maximum      |
 | minLength        | Waarde is korter dan minimale lengte {minLength}.         | minLength    |
-| maxLength        | Waarde is langer dan maximale lengte {minLength}.         | maxLength    |
-| bereik           | Lengte van waarde moet tussen {minimum} en {maximum} zijn.| size         |
```

Comment:
Ik neem aan dat deze validatie is verwijderd omdat hij nergens wordt uitgevoerd, klopt dat? 
Op zich terecht, alhoewel het een zinnige validatie kan zijn bij sommige parameters, maar toevoegen op het moment dat hij wel nodig is, is wat mij betreft prima.

Reply:
Het is nog anders, ik had `bereik` toegevoegd omdat ik hem dacht nodig te hebben, maar de bestaande `minLength` en `maxLength` lossen dat al op.